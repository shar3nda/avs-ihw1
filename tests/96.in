-7 -3 -10 5 5 -9 -5 -7 -3 4