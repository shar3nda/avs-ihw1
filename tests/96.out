0 2 -2 8 6 -2 9 2 10 5 -1